--- coq-8.0pl2/ide/undo.mli~	2005-12-10 02:08:33.000000000 +0100
+++ coq-8.0pl2/ide/undo.mli	2005-12-10 02:27:05.083226000 +0100
@@ -10,7 +10,7 @@
 
 (* An undoable view class *)
 
-class undoable_view : Gtk.text_view Gtk.obj ->
+class undoable_view : [> Gtk.text_view] Gtk.obj ->
 object
   inherit GText.view
   method undo : bool
--- coq-8.0pl2/ide/undo.ml~	2005-12-10 02:25:40.000000000 +0100
+++ coq-8.0pl2/ide/undo.ml	2005-12-10 02:26:07.331616750 +0100
@@ -18,7 +18,7 @@
   | Insert (s,i,l) -> Delete (s,i,l)
   | Delete (s,i,l) -> Insert (s,i,l)
 
-class undoable_view (tv:Gtk.text_view Gtk.obj) =
+class undoable_view (tv:[> Gtk.text_view] Gtk.obj) =
   let undo_lock = ref true in 
 object(self)
   inherit GText.view tv as super
