--- coq-8.1pl4/configure~	2008-10-09 11:40:41.000000000 +0200
+++ coq-8.1pl4/configure	2009-02-16 14:03:00.496249846 +0100
@@ -634,7 +634,7 @@
             # Beware of the final \r in Win32
             lablgtkdirtmp="$(ocamlfind query lablgtk2 2> /dev/null | tr -d '\r')"
             if [ "$lablgtkdirtmp" != "" ]; then
-                if [ -f "$lablgtkdirtmp/glib.cmi" -a -f "$lablgtkdirtmp/glib.mli" ]; then
+                if [ -f "$lablgtkdirtmp/glib.cmi" -o -f "$lablgtkdirtmp/glib.mli" ]; then
                     lablgtkdirfoundmsg="LabelGtk2 found by ocamlfind"
                     lablgtkdir=$lablgtkdirtmp
                     LABLGTKLIB=$lablgtkdir # Pour le message utilisateur
@@ -652,7 +652,7 @@
                 echo "$lablgtkdir is not a valid directory."
 		echo "Configuration script failed!"
 		exit 1
-	    elif [ -f "$lablgtkdir/glib.cmi" -a -f "$lablgtkdir/glib.mli" ]; then
+	    elif [ -f "$lablgtkdir/glib.cmi" -o -f "$lablgtkdir/glib.mli" ]; then
                 lablgtkdirfoundmsg="LablGtk2 directory found"
                 LABLGTKLIB=$lablgtkdir # Pour le message utilisateur
             else
@@ -664,9 +664,6 @@
     if [ "$lablgtkdir" = "" ]; then
 	echo "LablGtk2 not found: CoqIde will not be available."
 	COQIDE=no
-    elif [ -z "`grep -w convert_with_fallback "$lablgtkdir/glib.mli"`" ]; then
-	echo "$lablgtkdirfoundmsg but too old: CoqIde will not be available."
-	COQIDE=no;
     elif [ "$coqide_spec" = "yes" -a "$COQIDE" = "byte" ]; then 
 	echo "$lablgtkdirfoundmsg, bytecode CoqIde will be used as requested."
 	COQIDE=byte
@@ -855,15 +852,7 @@
 ####################################################
 
 if [ ! "$COQIDE" = "no" ]; then 
-    if grep "class view " "$lablgtkdir/gText.mli" | grep -q "\[>" ; then
-       if grep -q "?accepts_tab:bool" "$lablgtkdir/gText.mli" ; then
         cp -f ide/undo_lablgtk_ge212.mli ide/undo.mli
-       else
-	cp -f ide/undo_lablgtk_ge26.mli ide/undo.mli
-       fi
-    else
-	cp -f ide/undo_lablgtk_lt26.mli ide/undo.mli
-    fi
 fi  
 
 ##################################################
