--- coq-8.6/configure.ml~	2016-12-08 15:13:52.000000000 +0000
+++ coq-8.6/configure.ml	2017-06-05 23:04:32.311195087 +0000
@@ -669,8 +669,8 @@
     yell (sprintf "No such directory '%s' (%s)." dir msg)
   else if not (Sys.file_exists (dir/"gSourceView2.cmi")) then
     yell (sprintf "Incomplete LablGtk2 (%s): no %s/gSourceView2.cmi." msg dir)
-  else if not (Sys.file_exists (dir/"glib.mli")) then
-    yell (sprintf "Incomplete LablGtk2 (%s): no %s/glib.mli." msg dir)
+  else if not (Sys.file_exists (dir/"glib.cmi")) then
+    yell (sprintf "Incomplete LablGtk2 (%s): no %s/glib.cmi." msg dir)
   else true
 
 (** Detect and/or verify the Lablgtk2 location *)
@@ -701,7 +701,7 @@
 | Manual | Stdlib ->
   let test accu f =
     if accu then
-      let test = sprintf "grep -q -w %s %S/glib.mli" f dir in
+      let test = sprintf "grep -q -w %s %S/glib.cmi" f dir in
       Sys.command test = 0
     else false
   in
